src/hooks/v2/nftRewards/NFTRewardsToIPFS.ts

```diff
@@ -1,6 +1,6 @@
 import { NFTRewardTier } from 'models/v2/nftRewardTier'
 
-type CloudFunctionRewardTier = {
+export type CloudFunctionRewardTier = {
   edition: number
   isBooleanAmount: boolean
```

Comment:
API suggestion for @tankbottoms: perhaps we can use consistent casing, either snake_case or camelCase